src/Frame.h

```diff
@@ -390,6 +392,7 @@ class Frame :  public Obj {
 
 	trigger::TriggerPtr trigger;
 	const CallExpr* call;
```

Comment:
Might as well go ahead and initialize the other two variables here since you're initializing the new one.

Reply:
I did this for `call`.  Presumably `trigger` doesn't need it since `IntrusivePtr`s are initialized to nil.